External/HIP/builtin-logb-scalbn.hip

```diff
@@ -0,0 +1,58 @@
+
+#define __device__ __attribute__((device))
+
+#include <hip/hip_runtime.h>
+#include <cmath>
+#include <cstdio>
+
+__global__ void my_kernel(float a) {
```

Comment:
can we pass in an array and cover the following input values

16.0,  3.14,  0.0, -0.0,  INFINITY,  NAN

also, for the second argument of scalbn, cover 10, 0, -5